src/app/containers/Image/index.jsx

```diff
@@ -33,11 +33,16 @@ const ImageContainer = ({ blocks }) => {
     return null;
   }
 
-  const { locator, copyrightHolder } = rawImageBlock.model;
+  const { locator, originCode, copyrightHolder } = rawImageBlock.model;
   const altText = getText(altTextBlock);
   const copyright = getCopyright(copyrightHolder);
   const caption = getCaption(captionBlock);
-  const rawImageSrc = `https://ichef.bbci.co.uk/news/640${locator}`;
+  const rawImageSrc =
+    originCode === 'cpsprodpb'
+      ? `https://ichef.bbci.co.uk/news/640/${originCode}/${
```

Comment:
Can you please extract out this logic into a pure function and call that function here?  This would greatly simplify the logic that currently spans across lines 40-45. 

There are three cases: `originCode === 'cpsprodpb'`, `originCode === 'pips'`, `originCode === 'something else'` or `null`.
The logic here should be able to handle all three cases.